OidcDebugger/Startup.cs

```diff
@@ -34,7 +34,7 @@ public void ConfigureServices(IServiceCollection services)
             // Require HTTPS in production
             services.Configure<MvcOptions>(options =>
             {
-                options.Filters.Add(new RequireHttpsAttribute());
+                //options.Filters.Add(new RequireHttpsAttribute());
```

Comment:
I would take this out before finalizing the PR, but this is how I've been using it on my machine.